casual/casual-network/src/main/java/se/laz/casual/network/outbound/JEEConcurrencyFactory.java

```diff
@@ -62,4 +70,48 @@ public static ManagedExecutorService getManagedExecutorService()
         }
     }
 
+    /**
+     * Gets the default ManagedScheduledExecutorService on jboss. On some non-standard implementation
+     * it will fetch the default through java:comp/DefaultManagedScheduledExecutorService
+     * which normally shouldn't be accessible in this context.
+     * <p>
+     * If both of those fail a ScheduledExecutorService instance will be created
+     * through the standard java.util.concurrent.Executor which will share the
+     * same ScheduledExecutorService among all users.
+     *
+     * @return A ScheduledExecutorService instance.
+     */
+    public static ScheduledExecutorService getManagedScheduledExecutorService()
```

Comment:
getManagedScheduledExecutorService should perhaps be renamed since it can in fact return a plain vanilla ScheduledExecutorService?
Would geScheduledExecutorService suffice - the comment on the API tells the rest of the story?

Reply:
Equally, you could argue that in that unlikely event, we are "managing" access to it, as it will be reusable, so the naming is fine and the existing comment explains the situation well.

Reply:
Managed in the context is with regards to JEE though.
It's a minor thing though, my context is when using our stuff standalone outside of any appserver  ( Quarkus etc).

Reply:
I'll leave it as is for now. It's not an external api and can easily be renamed in the future if we find a better name that is both descriptive and doesn't infringe on JEE nomenclature